src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -90,36 +95,60 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
       val storage = Storage(client)
 
       val file = storage.createFile(
-          bucketId = "[BUCKET_ID]",
+          bucketId = "<BUCKET_ID>",
           fileId = ID.unique(),
-          file = File("./path-to-files/image.jpg"),
+          file = InputFile.fromPath("file.png"),
       )
   }
   ```
 
   ```http
-  POST /v1/storage/buckets/{bucketId}/files HTTP/1.1
-  Content-Type: multipart/form-data; boundary="cec8e8123c05ba25"
-  Content-Length: *Length of your entity body in bytes*
-  X-Appwrite-Project: [PROJECT_ID]
+      POST https://cloud.appwrite.io/v1/storage/buckets/<BUCKET_ID>/files HTTP/1.1
+      X-Appwrite-Project: <PROJECT_ID>
+      X-Appwrite-Mode: admin
+      X-Appwrite-Response-Format: 1.5.0
+      Cookie: COOKIE
+      Content-Type: multipart/form-data; boundary=---1234
+      Content-Length: LENGTH   
 
-  --cec8e8123c05ba25
-  Content-Disposition: form-data; name="operations"
+      -----1234
+      Content-Disposition: form-data; name="fileId"     
 
-  { "query": "mutation CreateFile($bucketId: String!, $fileId: String!, $file: InputFile!) { storageCreateFile(bucketId: $bucketId, fileId: $fileId, file: $file) { id } }", "variables": { "bucketId": "[BUCKET_ID]", "fileId": "[FILE_ID]", "file": null } }
-  --cec8e8123c05ba25
-  Content-Disposition: form-data; name="map"
+      <FILEID>
+      -----1234
+      Content-Disposition: form-data; name="file"; filename="text.png"
+      Content-Type: image/PNG
 
-  { "0": ["variables.file"] }
-  --cec8e8123c05ba25
-  Content-Disposition: form-data; name="0"; filename="file.txt"
-  Content-Type: text/plain
+< ./miku.png
 
-  File content.
+-----1234
 
-  --cec8e8123c05ba25--
   ```
+  ```client-react-native
+  import { Client, Storage, ID } from 'react-native-appwrite';
 
+  const client = new Client()
+      .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+      .setProject('<PROJECT_ID>');                 // Your project ID
+
+  const storage = new Storage(client);
+
+  const promise = await storage.createFile(
+      '<BUCKET_ID>',
+      ID.unique(),
+      { 
+        name: 'file.jpeg', 
+        type: 'image/jpeg', 
+        uri: 'file://user/text.jpeg', 
+        size: 183799 
+  });
```

Comment:
Indentation is 4 spaces so keep that consistent. And let's put the closing parenthesis on it's own line to separate the parameters from the parenthesis.

```suggestion
      { 
          name: 'file.jpeg', 
          type: 'image/jpeg', 
          uri: 'file://user/text.jpeg', 
          size: 183799 
    }
);
```